### DISCUSSION TITLE
late-lessons-academia/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#15] [Announcements]
# late-lessons-academia/

تو دوران ارشد مطالبی رو به طور غیرمستقیم یاد گرفتم که فکر می‌کنم اگر از اول می‌دونستم، شاید خیلی متفاوت عمل می‌کردم.
مثل همیشه، فکر نمی‌کنم لازم باشه که تأکید کنم که این حرف‌ها وحی منزل نیستند و بسته به شرایط هر فرد ممکنه تغییر کنند و قص علی هذا.
کار تیمی به نظرم مهم‌ترین چیزی که تو ارشد یاد گرفتم، اهمیت کار تیمی هست. تیمی کارکردن از موضوعی که روش کار می‌کنید، هم مهم‌تر هست.

https://aprd.ir/late-lessons-academia/

**Comment by HashemAlavi** [comment]
سلام امیر عزیز🌹

مرسی بابت معرفی سایت‌ها. راستش من عاشق آرشیو کردنم. تقریباً چند 10 تا سایت توی بوکمارک مرورگرم دارم. اینا هم بهشون اضافه شد:)

نگاه بلند مدتی هم که توضیحش رو دادی برای من تذکر به جایی بود. منم گاهی اوقات یک سری افکاری به ذهنم میرسه که خب که چی؟ مثلاً درس میخونی نهایتش میخوای بری تو داروخونه دیگه اونجا چی واقعاً به ملت اضافه میشه؟ بهتر نیست بیخیال دانشگاه بشی و بری سراغ کار؟ لااقل تو بازار کار یک کار مفیدی وجود داره.
و بعد از کلّی درگیری ذهنی و گاهاً کلافه شدن و سردرد گرفتن و سوزش معده بیخیال میشم و میرم سراغ کارایی که باید انجام بدم. میگم ولش کن هر چی شد شد. من که قرار نیست به تنهایی کار خاصی انجام بدم.

بازم ممنون از نوشته‌هات. مایه دلگرمیه:)

**Comment by najmehabdolinasab** [comment]
سلام اقای برومند طی سرچی که اخیرا راجع به موضوع پایان نامه ام داشتم متوجه شدم کاری که شما به عنوان پایان نامه پرزنت کردید بسار نزدیک به موضوع من هست امکانش هست 
لطف کنید در جهت پیش برد این مسیر بنده رو راهنمایی بفرمایید؟ متاسفانه صفحه چت خصوصی پیدا نکردم برای مطرح کردن ابهاماتم
